pkgs/by-name/he/hellmaker/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hell-maker";
+  version = "unstable-2023-03-18";
+
+  src = fetchFromGitHub {
+    owner = "0xNinjaCyclone";
+    repo = "hellMaker";
+    rev = "2e9efe2aed9412f67a8606544392a2df10d3a2d0";
+    hash = "sha256-zbtzlYGCLW/lt7GJvMHao/MZhdghNBQCQsjUImL1RC4=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/hellmaker}
+    cp -R * $out/share/hellmaker
+    makeWrapper ${python3.interpreter} $out/bin/hellmaker \
+      --set PYTHONPATH "$PYTHONPATH:$out/share/hellmaker/hellMaker.py" \
+      --add-flags "$out/share/hellmaker/hellMaker.py"
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Avoid nested `with`.

---

Are you going to do this change?

---

changes have been applied.